Satellite-im/UplinkWeb
# chore(updates): minor update

### What this PR does 📖

- Removes not needed commented out code
- Removes not used imports
- Format code
- Fix warning on vscode because was missing family font on `src/app.scss`
<img width="1135" alt="Captura de ecrã 2024-11-20, às 23 04 57" src="https://github.com/user-attachments/assets/eade6244-495b-425c-a8d9-fc494c4dd19b">


How to test:
- go to vscode
- search for `src/app.scss` file
- this yellow warning should not appear

before

<img width="294" alt="Captura de ecrã 2024-11-20, às 23 09 00" src="https://github.com/user-attachments/assets/39b23acb-e8c8-4fd8-9b17-00ff72607b4a">

after

<img width="332" alt="Captura de ecrã 2024-11-20, às 23 09 24" src="https://github.com/user-attachments/assets/e28e815f-d360-4ae3-b70d-6a53bd68efbc">



- Adds title on `src/app.html`

before

<img width="312" alt="Captura de ecrã 2024-11-20, às 22 35 11" src="https://github.com/user-attachments/assets/d110b16c-5c72-448f-992d-5c6977edfb7b">

after

<img width="293" alt="Captura de ecrã 2024-11-20, às 22 35 16" src="https://github.com/user-attachments/assets/c2045454-497f-4c68-870a-7940d5bf45e6">
